### PR TITLE
Feature lwdev 10409 blockchain explorers

### PR DESCRIPTION
**Pull request by OttoVT**

### CODE REVIEW

File: LkeServices/Blockchain/BlockchainExplorersProvider.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                            ?.Replace(Lykke.Service.BlockchainSettings.Contract.Constants.TxHashTemplate, txHash)
+                    };
+
+                    return explorer;
+                });
+
+            return mapped;
+        }
+
+        public async Task InvalidateCacheAsync()
```
Comment (KonstantinRyazantsev) [inline LkeServices/Blockchain/BlockchainExplorersProvider.cs]: Unused method. And why someone who needs to invalidate the cache just not use IClientCacheManager?

File: LkeServices/Blockchain/BlockchainExplorersProvider.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        public BlockchainExplorersProvider(Lykke.Service.BlockchainSettings.Client.IBlockchainSettingsClient client,
+            IClientCacheManager clientCacheManager)
+        {
+            _client = client;
+            _clientCacheManager = clientCacheManager;
+        }
+
+        public async Task<IEnumerable<BlockchainExplorerLink>> GetAsync(string blockchainType, string txHash)
+        {
+            var explorers = await _client.GetBlockchainExplorerByTypeAsync(blockchainType);
```
Comment (KonstantinRyazantsev) [inline LkeServices/Blockchain/BlockchainExplorersProvider.cs]: Is this call cached?

Comment (OttoVT) [reply LkeServices/Blockchain/BlockchainExplorersProvider.cs]: image

File: LkeServices/Blockchain/BlockchainExplorersProvider.cs (hunk 1 above)
Comment (OttoVT) [reply LkeServices/Blockchain/BlockchainExplorersProvider.cs]: fixed

**Pull request merged**